internal/cli/servercmd/server.go

```diff
@@ -117,6 +117,18 @@ func (c *Config) Exec(ctx context.Context, args []string) error {
 			return errors.Wrap(err, "failed to set up registry")
 		}
 		registry = reg
+	case "gcs":
+		if c.S3Bucket == "" {
+			return errors.Wrap(flag.ErrHelp, "missing flag -s3-bucket")
```

Comment:
To avoid confusion I would prefer to use a dedicated flag for gcs here, to keep things explicit.

Reply:
We surly can. However as S3 and GCS rely on the bucket parameter we might want to switch to `-bucket` instead? Its a matter of tase I guess.

Reply:
I introduced the gcs related parameters.